.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# How to become a contributor and submit your own code
```

Comment:
*sighs* so running `nodejs-repo-tools` has a side effect of creating/replacing a lot of these files 😟  Can you delete this one?